src/vm/diagnosticsprotocol.h

```diff
@@ -210,7 +210,7 @@ namespace DiagnosticsIpc
 
         // Initialize an outgoing IpcMessage with a header and payload
         template <typename T>
-        bool Initialize(IpcHeader header, T&& payload)
+        bool InitializePayload(IpcHeader header, T&& payload)
```

Comment:
I think it would be better to delete this method and just use the other overload.

Reply:
That was my reaction too.

Code is sending a S_OK constant as in src/vm/dumpdiagnosticprotocolhelper.cpp below. I couldn't get that to compile Any suggestions?

Reply:
HRESULT success = S_OK;
... pass success to the method ...

Reply:
will do, thanks